src/app/ui/user-profile/user-profile.component.html

```diff
@@ -49,11 +61,23 @@
           <mat-card-subtitle>{{ otherUser.membership?.teamName || 'No Team Yet' }}</mat-card-subtitle>
         </mat-card-header>
         <mat-card-content>
-          <span *ngIf="otherUser.membership?.isLead">
-            <mat-icon>work</mat-icon>
-            <b>Team Lead</b>
-          </span>
-          <span><mat-icon>email</mat-icon> {{ otherUser.email }}</span>
+          <div class="player-info">
+            <div class="basic-info">
+              <span *ngIf="otherUser.membership?.isLead">
+                <mat-icon>work</mat-icon>
+                <b>Team Lead</b>
+              </span>
+              <span><mat-icon>email</mat-icon> {{ otherUser.email }}</span>
+            </div>
+            <div class="player-stats">
+              <span class="exp-pts">
+                {{ otherUser.totalExp }}
```

Comment:
not sure if we wanted here totalXp or just seasonXP, lets confirm with them 1st. 
choosing only one of the 2 will save us read transactions.
also it might be better to use xp suffixed properties so that it would be in uniform with their labels